openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMModelCreator.java

```diff
@@ -237,6 +245,17 @@ public LightGBMBinaryClassificationModel loadModel(final Path modelPath,
             throw new ModelLoadingException(ERROR_MSG_SCHEMA_WITH_WRONG_PREDICTIVE_FIELDS_SIZE);
         }
 
+        final String[] schemaFeatureNames = getFeatureNamesFrom(schema);
+
+        if (!Arrays.equals(model.getBoosterFeatureNames(), schemaFeatureNames)) {
+
+            logger.error("Schema with wrong predictive fields names: {} - Expected: {}",
+                    String.join(", ", schemaFeatureNames),
+                    String.join(", ", model.getBoosterFeatureNames()));
```

Comment:
Besides the 2 "CSV" lines, shouldn't we print the index and expected and actual feature name where there's a mismatch?

Imagine the user has 300 features (not uncommon), we should help them find which field is wrong right?

Reply:
I agree, 'll switch to manual comparison.

Reply:
please review this.